app/auth/routes.py

```diff
@@ -75,7 +136,7 @@ def user_by_name(id=None,username=None):
         }), 400
     if request.method == 'GET':
         try:
-            return jsonify(user)
+            return jsonify(user.serialize())
```

Comment:
Making an instance json is different than fetching it from db. So is better to have User.find_by_id() which will return a User Object and then user.serialize() which will serialize this object to a json one